components/automate-ui/src/app/modules/policy/details/policy-details.component.ts

```diff
@@ -20,6 +20,10 @@ import {
 
 export type PolicyTabName = 'definition' | 'members';
 
+export interface KeyboardEvent {
+  isUserInput: boolean;
+}
+
```

Comment:
By default - isUserInput is not part of the KeyboardEvent interface - this extends it.

Reply:
`isUserInput` is provided by Angular Material.  Every time an option is selected there is an event fired for selected and an event fired for deselected.  The selected event passed through `isUserInput = true` and the deselection event passes through `isUserInput = false`;